neuralmonkey/encoders/numpy_stateful_filler.py

```diff
@@ -23,14 +28,21 @@ def __init__(self,
                  save_checkpoint: str = None,
                  load_checkpoint: str = None,
                  initializers: InitializerSpecs = None) -> None:
+        """Instantiate StatefulFiller.
+
+        Args:
+            name: Name of the model part.
+            dimension: Dimensionality of the input.
+            data_id: Series containing the numpy objects.
```

Comment:
Chápal bych, kdybys zdokumentoval jen věci, který to nedědí od `ModelPart`. Ale jednak tu máš `name`, chybí ty checkpointy a initializéry, ale chybí i `output_shape`, kterej v `ModelPart` neni.

Reply:
Vzhledem k budoucí pydocstylizaci bych sem narval všechno.

Reply:
Rychlej google našel jen složitý řešení pro děďbu docstringů

Reply:
Name je demence vynchávat protože je to první argument. Psát všude chcekpoints a initializer mi přijde blbost, protože až se to změní, tak zapomeneme všude nakopírovat změny do všech doctringů a bude v tom akorát bordel.

Reply:
No ale output shape to přidat chce

Reply:
Ten jsem přidal